### PR TITLE
Update config to support newer polymer.json build properties

### LINKED ISSUE #346: images folder not included after running polymer build
### Versions & Environment

- Polymer CLI: 0.13
- node: v4.4.7
- Operating System: Mac OSx 10.11.6

#### Steps to Reproduce

1. Create an application project: `polymer init application`
2. Build: `polymer build`

#### Expected Results

The folder  build/bundled/images/ should be present

#### Actual Results

The folder  build/bundled/images/ is not present

### PR DESCRIPTION
**Pull request by FredKSchott**
Fixes #346 #376

~~WIP, still needs tests!~~

/cc @justinfagnani @robdodson

### CODE REVIEW

**Comment by FredKSchott** [comment]
@justinfagnani tests updated, ptal

File: src/project-config.ts [hunk 1]
```diff
@@ -14,12 +14,27 @@ import * as logging from 'plylog';
...
+];
+
 export interface ProjectConfigOptions {
   root?: string;
   entrypoint?: string;
   shell?: string;
   fragment?: string[];
   fragments?: string[];
+
+  // @NOTE(fks) 08-23-2016: The options below were forked by polymer-build.
```
Comment (justinfagnani) [inline src/project-config.ts]: what's `@NOTE`?

Comment (FredKSchott) [inline src/project-config.ts]: Whoops, old habits. That's something I've used in past projects. 

fwiw tagging a comment with a date and person can be super useful for comments related to tech-debt or that are time sensitive that someone may want to follow up on months/years later.

Comment (justinfagnani) [inline src/project-config.ts]: I just use TODOs, since most IDEs and a lot of other tools support it

File: test/config/polymer.json [hunk 2]
```diff
@@ -2,5 +2,9 @@
   "entrypoint": "foo.html",
   "fragments": [
     "bar.html"
+  ],
```
Comment (justinfagnani) [inline test/config/polymer.json]: test includeDependencies too?

**Comment by justinfagnani** [comment]
LGTM

File: test/config/polymer.json (hunk 2 above)
Comment (FredKSchott) [inline test/config/polymer.json]: done

File: src/project-config.ts (hunk 1 above)
Comment (FredKSchott) [inline src/project-config.ts]: yea I guess that could be considered a TODO as well

**Comment by abdonrd** [comment]
Yay! Thanks! 🎉

**Pull request merged**